crates/typst-syntax/src/lexer.rs

```diff
@@ -919,7 +999,7 @@ pub fn is_id_start(c: char) -> bool {
 /// Whether a character can continue an identifier.
 #[inline]
 pub fn is_id_continue(c: char) -> bool {
-    is_xid_continue(c) || c == '_' || c == '-'
+    is_xid_continue(c) || c == '-'
```

Comment:
Does this not affect other parser code?

Reply:
I forgot to make a note of this, but '_' is already a continuing character in the Unicode Standard. (I used [this](https://util.unicode.org/UnicodeJsps/list-unicodeset.jsp?a=[:XID_Continue=Yes:]), and it includes `_ U+005F LOW LINE`, so it isn't actually necessary I think). So no, it doesn't affect anything.

Reply:
I think I kept it intentionally to mirror is_id_start

Reply:
Would you prefer adding it back then?

Reply:
Yeah, I think it makes it so that one stumbles less upon seeing this code.